scripts/test.sh

```diff
@@ -20,7 +20,7 @@ echo '''
                                       Nose
 ================================================================================
 '''
-nosetests . -v --with-coverage --cover-erase
```

Comment:
Generally, we always need to know which line is covered or else in local environment.